localisation/replace/odessus_replace_l_english.yml

```diff
@@ -0,0 +1,3 @@
+﻿l_english:
+ timariot_system:0 "Timar System"
+ timariot_system_desc:1 "The Ottoman Sultanate developed a system of feudal land holding to build up and maintain a cavalry force. Each Timariot was responsible for supplying cavalry for the imperial army based upon the amount of land he held."
```

Comment:
i didnt tell you this yet: those numbers after the colon only matter to paradox employees and modders usually remove/omit them